src/krati_Jain/Assignment_30/SecondMaxNumber.java

```diff
@@ -0,0 +1,32 @@
+/*WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55 */
+
+package krati_Jain.Assignment_30;
+
+public class SecondMaxNumber {
+
+	public static void main(String[] args) {
+		int[] inputArr = { 10, 33, 43, 55, 97, 11, 3, 100, 1000, 0};
+		SecondMaxNumber secondMaxNum = new SecondMaxNumber();
+		secondMaxNum.findSecondMax(inputArr);
+	}
+
+	void findSecondMax(int[] intArry) {
+		int holder = 0;
+		for (int index = 0; index < intArry.length; index++) { // {10,33,43,55,97,11,3, 100, 1000, 0} 10,33
```

Comment:
You can handle with single loop, not require two loop,

Take look of below code
Shashank - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1560/files